supporting_scripts/analysis-of-endpoint-connections/analysisOfEndpointConnections.config.yml

```diff
@@ -0,0 +1,282 @@
+  - 'api/exercises/{exerciseId}/feedback-details'
+  - 'api/files/user/profile-pictures/{userId}/*'
+  - 'api/account'
+  - 'api/account/ssh-public-key'
+  - 'api/account/user-vcs-access-token'
+  - 'api/account/participation-vcs-access-token'
+  - 'api/public/account'
+  - 'api/public/account/change-language'
+  - 'api/public/saml2'
+  - 'api/public/logout'
+
+  # Issue: #9372 and #9371
+  - 'api/courses/{courseId}/students'
+  - 'api/courses/{courseId}/tutors'
+  - 'api/courses/{courseId}/editors'
+  - 'api/courses/{courseId}/instructors'
+  - 'api/courses/{courseId}/title'
+  - 'api/courses/{courseId}/students/{studentLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/tutors/{tutorLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/editors/{editorLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/instructors/{instructorLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/students/{studentLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/tutors/{tutorLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/editors/{editorLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/instructors/{instructorLogin: + Constants.LOGIN_REGEX + }'
+  - 'api/courses/{courseId}/exams/{examId}/students/{studentLogin: + Constants.LOGIN_REGEX + }/attendance-check'
+  - 'api/programming-exercises/{exerciseId} + /static-code-analysis-categories/import'
+
+  # Issue: #9371
+  - 'api/athena/text-exercises/{exerciseId}/submissions/{submissionId}/feedback-suggestions'
+  - 'api/athena/programming-exercises/{exerciseId}/submissions/{submissionId}/feedback-suggestions'
+  - 'api/athena/modeling-exercises/{exerciseId}/submissions/{submissionId}/feedback-suggestions'
+  - 'api/athena/courses/{courseId}/text-exercises/available-modules'
+  - 'api/athena/courses/{courseId}/programming-exercises/available-modules'
+  - 'api/athena/courses/{courseId}/modeling-exercises/available-modules'
+  - 'api/quiz-exercises/{quizExerciseId}/{action}'
+
+
+  - 'api/public/athena/programming-exercises/{exerciseId}/submissions/{submissionId}/repository'
+  - 'api/public/athena/programming-exercises/{exerciseId}/repository/template'
+  - 'api/public/athena/programming-exercises/{exerciseId}/repository/solution'
+  - 'api/public/athena/programming-exercises/{exerciseId}/repository/tests'
+  - 'api/programming-exercises/{exerciseId}/build-plan/for-editor'
+  - 'api/programming-exercises/{exerciseId}/build-plan'
+  - 'api/build-log/{buildJobId}'
+  - 'api/exercises/{exerciseId}/submissions-for-import'
+  - 'api/courses/{courseId}/answer-posts'
+  - 'api/courses/{courseId}/answer-posts/{answerPostId}'
+  - 'api/courses/{courseId}/posts'
+  - 'api/courses/{courseId}/posts/{postId}'
+  - 'api/lectures/{lectureId}/attachment-units/{attachmentUnitId}'
+  - 'api/course-competencies/{competencyId}/title'
+  - 'api/courses/{courseId}/competencies'
```

Comment:
Why are these endpoints excluded? Some of them are definitely used.
At least add a comment above this section explaining it.